email_recovery/lib/email_recovery/converter/email_recovery_action_id_converter.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:email_recovery/email_recovery/email_recovery_action_id.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+class EmailRecoveryActionIdConverter implements JsonConverter<EmailRecoveryActionId, String> {
```

Comment:
If `id=null` then `EmailRecoveryActionIdConverter ` does not work correctly. Should be support nullable